presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java

```diff
@@ -92,9 +92,9 @@ public class NativeExecutionSystemConfig
     private int systemMemoryGb = 10;
     private DataSize queryMemoryGb = new DataSize(systemMemoryGb, DataSize.Unit.GIGABYTE);
     private boolean useMmapAllocator = true;
-    private boolean enableMemoryArbitration = true;
-    private long memoryPoolInitCapacity = 512 << 20;
-    private long memoryPoolTransferCapacity = 256 << 20;
+    private String memoryArbitratorKind = "SHARED";
+    private long memoryPoolInitCapacity = 10 << 30;
```

Comment:
Do we want to leave a bit for pre-fetch?

Reply:
Yeah, we could consider adding a config for the prefetch memory.